invenio_app_ils/ui/src/invenio_app_ils/forms/components/CountryField.js

```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { VocabularyField } from '../core';
 import { Dropdown, Flag } from 'semantic-ui-react';
+import { names as FLAG_NAMES } from 'semantic-ui-react/dist/commonjs/elements/Flag/Flag';
```

Comment:
@equadon @kprzerwa this is the only way I found to avoid the list.
Given that there are no future releases (kind of dead project), I guess it is pretty safe... And also if the path is not found, there is a build error...
They exported values in TypeScript but in JS :(

What do you think?

Reply:
Thank you! I tried to make that work but wasn't successful. It's not optimal but given how the exports work we don't have many choices.